File: Apps/iOS/LoyalAZ/LoyalAZ/LoyalAZ.m
//
//  LoyalAZ.m
//  TestXML
//

#import "LoyalAZ.h"

@implementation LoyalAZ
@synthesize programs,user;
@synthesize sync;
@synthesize find_enable;
@synthesize enableFBPost;
@synthesize coupons;
@synthesize ads_enable;
//@synthesize methods;

- (id)init
{
    self = [super init];
    if (self) {
        
        programs = [[NSMutableArray alloc]init];
        coupons = [[NSMutableArray alloc]init];
        //methods = [[NSMutableArray alloc]init];
    }
    else {
        user = [[User alloc]init];
    }
    return self;
}


@end

File: Apps/Android/LoyalAZ/src/com/outstandingresults/DataObjects/LoyalAZ.java
package com.outstandingresults.DataObjects;



import java.util.List;

public class LoyalAZ {

public String sync;
public User User;
public List<Program> programs;
public String find_enable;
public String enableFBPost;
public List<Coupon> coupons;
public String ads_enable;
	public LoyalAZ()
	{
		sync = "";
		find_enable = "";
		enableFBPost = "";
		User = new User();
		ads_enable = "";
	}
	
}
